composer.json

```diff
@@ -22,7 +22,8 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "laminas/laminas-dependency-plugin": false
```

Comment:
```suggestion
```
Didn't we already remove this?

Reply:
a phpunit check failed related to this, so i had to update 'allow-plugins'
https://github.com/dotkernel/dot-errorhandler/actions/runs/9159744416/job/25180860644

Reply:
removed in the latest commit
along with php 8.1